webapp/view/ObjectPage.view.xml

```diff
@@ -0,0 +1,202 @@
+<mvc:View
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+    xmlns:u="sap.uxap"
+    xmlns:form="sap.ui.layout.form"
+	xmlns:core="sap.ui.core"
+	controllerName="products.app.controller.ObjectPage">
+
+
+    <u:ObjectPageLayout 
+        class="sapUiNoContentPadding">
+        <u:headerTitle>
+            <u:ObjectPageHeader objectTitle="{Name}" /> 
+        </u:headerTitle>
+      
+        <u:headerContent>
+            <VBox>
+				<ObjectStatus title="{i18n>DescriptionText}" text="{Description}" />
+				<ObjectStatus 
+                    title="{i18n>PriceText}" 
+                    text="{
+                        parts: [
+                            'i18n>PriceAndCurrencyText',
+                            'Price',
+                            'Currency'
+                        ],
+                        formatter: '.formatMessage'
+                    }" 
+                />
+				<ObjectStatus title="{i18n>QuantityText}" text="{Quantity}" />
+			</VBox>
+            
+            <ObjectStatus 
+                title="{i18n>CategoryText}" 
+                text="{
+                    path: 'Category',
+                    formatter: '.getCategoriesName'
+                }"
+            />
+            <ObjectStatus 
+                title="{i18n>AvailabilityText}" 
+                text="{
+                    parts: [
+                        'i18n>PriceAndCurrencyText',
+                        'Price',
+                        'Currency'
+                    ],
+                    formatter: '.formatMessage'
+                }"
+            />
+            <ObjectStatus 
+                title="{i18n>CreatedAtText}" 
+                text="{
+                    path: 'CreatedAt',
+                    type: 'sap.ui.model.type.Date',
+                    formatOptions: {
+                        source: {
+                            pattern: 'yyyy-MM-ddTHH:mm:ss'
```

Comment:
This is not quite good because people from different countries see dates differently.
Please remove it and find another way how to format date.